.gitignore

```
@@ -30,5 +30,5 @@ powsybl.log
 # Maven wrapper jar
 .mvn/wrapper/*.jar
 
-# Generated readthedocs pages
-build-docs/
+# Documentation
+/docs/_build
```

It may be better to keep both, when switching between old and new branches, you can still have the old generated doc directory.

```suggestion
build-docs/
/docs/_build
```